gem/styles/gem-springster/_base.scss

```diff
@@ -1,8 +1,6 @@
 html, body  {
```

Comment:
I think the base font-size = 16px set here helps to using various size unit either %, em and etc.